### PR TITLE
DBZ-3115 Add support for JSON column to Vitess 9.0.0

### PR DESCRIPTION
**Pull request by keweishang**
https://issues.redhat.com/browse/DBZ-3115

### CODE REVIEW

File: documentation/modules/ROOT/pages/connectors/vitess.adoc [hunk 1]
```diff
@@ -773,7 +773,7 @@ a|_n/a_
 a|_n/a_
 
 |`JSON`
-|Unsupported yet
+|`STRING`
```
Comment (gunnarmorling) [inline documentation/modules/ROOT/pages/connectors/vitess.adoc:776]: I suppose it is exported as the `io.debezium.data.Json` logical type?

Comment (keweishang) [reply documentation/modules/ROOT/pages/connectors/vitess.adoc:776]: That's a good point. Unfortunately, Vitess Connector doesn't export any `io.debezium` semantic type (schema name) yet, same for other column types. 

I'll try and add `io.debezium.data.Json` semantic type (schema name) to the `JSON` column type in the corresponding PR: https://github.com/debezium/debezium-connector-vitess/pull/19

**Comment by keweishang** [comment]
This PR is ready to be reviewed

**Comment by gunnarmorling** [comment]
Applied; thx, @keweishang!

**Pull request merged**